ScalaApp/settings.py

```diff
@@ -163,3 +164,34 @@
 
 # Include Scala settings
 from ScalaApp.scala_settings import *
+
+# Enable OpenID Connect
+AUTHENTICATION_BACKENDS = [
+    'django.contrib.auth.backends.ModelBackend',
+    'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
+]
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler',
+        },
+    },
+    'loggers': {
+        'mozilla_django_oidc': {
+            'handlers': ['console'],
+            'level': 'DEBUG'
+        },
+    },
+}
+
```

Comment:
Is dit niet netter in een ander bestand wat dit bestand extend? Vergelijkbaar met scala_settings.py. Vooral wanneer ook andere verenigingen open-id inlogsystemen hebben/krijgen is dit handig.